openmdao/visualization/n2_viewer/style/icon.css

```diff
@@ -51,115 +51,103 @@ Instructions on editing the icons used for the toolbar.
 	/* Better Font Rendering =========== */
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
-}
-
-.icon-question-circle:before {
-	content: '\e90d';
-}
-
-.icon-bezier-curve:before {
-	content: '\e908';
-}
-
-.icon-caret-right:before {
-	content: '\e909';
-}
+  }
 
-.icon-chevron-left:before {
-	content: '\e90a';
-}
+  .icon-backward:before {
+	content: "\e900";
+  }
 
-.icon-info-circle:before {
-	content: '\e90b';
-}
+  .icon-bezier-curve:before {
+	content: "\e901";
+  }
 
-.icon-key:before {
-	content: '\e90c';
-}
+  .icon-caret-right:before {
+	content: "\e902";
+  }
 
-.icon-save:before {
-	content: '\e90e';
-}
+  .icon-chevron-left:before {
+	content: "\e903";
+  }
 
-.icon-forward:before {
-	content: '\e906';
-}
+  .icon-collapse-depth:before {
+	content: "\e904";
+  }
 
-.icon-forward {
-	font-size: 13px !important;
-}
+  .icon-enlarge:before {
+	content: "\e905";
+  }
 
-.icon-backward:before {
-	content: '\e907';
-}
+  .icon-enlarge2:before {
+	content: "\e906";
+  }
 
-.icon-backward {
-	font-size: 13px !important;
```

Comment:
The back and forward toolbar buttons got huge - Looks like we need to move the `.icon-forward` and `.icon-backward` defs out of icon.css since we expect it to be autogenerated by the icomoon site. Probably into toolbar.css.